honeycombio/resource_trigger_test.go

```diff
@@ -208,6 +218,48 @@ resource "honeycombio_trigger" "test" {
 }`, dataset, dataset, frequency)
 }
 
+func testAccTriggerConfigWithAlertType(dataset string, alertType string) string {
```

Comment:
I don't think this gets called? Cool to just rely on what you've done above and remove it

Reply:
Alternatively, you can add it in as a test step above and test with both `on_change` and `on_true`

Reply:
ah yep, the test is not used - this was from my initial first pass. can remove in lieu of the validation written in